index.js

```diff
@@ -27,16 +27,24 @@ var hooks = [
   'update'
 ];
 
+function findGitRoot() {
+  var gitRevParseTopLevel = exec('git rev-parse --show-toplevel');
+  if (gitRevParseTopLevel.code !== 0) {
+    console.error('guppy-cli needs a git repository to work with.');
+    return exit(1);
```

Comment:
Since this is the lib meant to be consumed by other packages (including the `bin/index.js`), we should return an error to expose it to the caller, and let the caller handle the error (you can `exit(1)` there), rather than doing it here.